src/libs/state.ts

```diff
@@ -5,6 +5,9 @@ import type { WorkspaceWrapper } from 'src/libs/workspace-utils';
 
 export const routeHandlersStore = Utils.atom<unknown[]>([]);
 
+// This is naughty - casting to `any` is ultimately not what we want, but while converting to TypeScript,
+// we don't want to cascade into migrating every file all at once,
+// so sometimes this is the answer to satisfy the TS compiler
```

Comment:
```suggestion
```

I don't think this comment is really necessary. While the code base is in transition from JS to TS, I think is fairly widely understood and there are many `any`s in this file and elsewhere.